<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q0CryomoduleSelector</class>
 <widget class="QWidget" name="Q0CryomoduleSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Q0 Cryomodule Selection</string>
  </property>
  <property name="windowFilePath">
   <string>cmSelection.ui</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>721</width>
        <height>1052</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="PyDMEmbeddedDisplay" name="cryomodulesL0B">
         <property name="toolTip">
          <string/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="macros" stdset="0">
          <string>area=L0B</string>
         </property>
         <property name="filename" stdset="0">
          <string>areaCryomoduleSelector.ui</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PyDMEmbeddedDisplay" name="cryomodulesL1B">
         <property name="toolTip">
          <string/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="macros" stdset="0">
          <string>area=L1B</string>
         </property>
         <property name="filename" stdset="0">
          <string>areaCryomoduleSelector.ui</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PyDMEmbeddedDisplay" name="cryomodulesL2B">
         <property name="toolTip">
          <string/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="macros" stdset="0">
          <string>area=L2B</string>
         </property>
         <property name="filename" stdset="0">
          <string>areaCryomoduleSelector.ui</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PyDMEmbeddedDisplay" name="cryomodulesL3B">
         <property name="toolTip">
          <string/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="macros" stdset="0">
          <string>area=L3B</string>
         </property>
         <property name="filename" stdset="0">
          <string>areaCryomoduleSelector.ui</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMEmbeddedDisplay</class>
   <extends>QFrame</extends>
   <header>pydm.widgets.embedded_display</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
